concepts/05 Widgets/CheckBox/00 Overview.md

```diff
@@ -42,6 +42,50 @@ The following code adds the **CheckBox** to your page.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxCheckBox
+            text="Check me"
+            :value="undefined"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxCheckBox from 'devextreme-vue/check-box';
+
+    export default {
+        components: {
+            DxCheckBox
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import CheckBox from 'devextreme-react/check-box';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <CheckBox
+                    text="Check me"
+                    value={undefined}
```

Comment:
Bind this value to a variable and handle changes to imitate two-way binding

Reply:
fixed